pkg/services/object/put/validation.go

```diff
@@ -55,6 +57,20 @@ func (t *validatingTarget) WriteHeader(obj *objectSDK.Object) error {
 			return ErrExceedingMaxSize
 		}
 
+		if t.homomorphicChecksumRequired {
```

Comment:
What if we already have some objects like that in our networks?

Reply:
then they'll be considered invalid, and i think its ok: i doubt anybody could slice objects on his own, at the same time all nodes up to v0.37 worked correctly. So, i doubt there are objects w/o homomorphic checksums (if container requires). Anyway, this check is needed to control container policy compliance

Reply:
I fear we'll have replication problems. I agree that it's a correct check, but if we have some objects like that already, it'd be a problem.

Reply:
i don't think we have such objects. But even if so - what can we do? I think nothing

Reply:
We must keep them since they were accepted at some point.

Reply:
keep - they will be stored even if replication fails. But they wont move within container, and afaik we can do nothing with this (hard to diff `Replicate` from `Put`) 

again, i don't think such objects exist in current networks

Reply:
Can we check for it? Walk through objects with some tool and see if it's the case?

Reply:
there is no such tool

Reply:
upd: now we have one d85954cd0c0d5949f7b3424a48ebc8923807f03a

not planned to be merged, but pushed to be observable

@roman-khimov is that enough? it'll require per-node entrance

Reply:
Really nice, no problematic objects identified, so we're safe. Let's leave the tool in git history.

Reply:
done